astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -2702,6 +2703,29 @@ def test_vector_norm(self):
             assert_array_equal(n2, expected2)
 
 
+@pytest.mark.skipif(NUMPY_LT_2_3, reason="np.matvec and np.vecmat are new in NumPy 2.3")
```

Comment:
Sorry to be annoying, but this should be in `test_quantity_ufunc` (gufuncs are there too...), with `matmul` and `vecdot`

Reply:
no annoyance at all. Thanks for your attention to details !